aks/application/resources.tf

```diff
@@ -1,7 +1,10 @@
 locals {
   name_suffix = var.name != null ? "-${var.name}" : ""
+  app_name    = "${var.service_name}-${var.environment}${local.name_suffix}"
 
-  app_name = "${var.service_name}-${var.environment}${local.name_suffix}"
+  http_probe_enabled = var.is_web && var.probe_path != null
```

Comment:
This will always be true for webapps if probe_path default value is "/healthcheck"
We may need another boolean input var "http_probe_enabled"

Reply:
This conditional is only to consider the case when the `probe_path` has been set to `null` for whatever reason - it may never happen but good to be safe I'd say.